README.md

```diff
@@ -1 +1,160 @@
-# Alton
\ No newline at end of file
+# Alton
+
+- [1. Introduction](#1-introduction)
+- [2. How to use](#2-how-to-use)
+
+## 1. Introduction
+
+Alton is a piece of middleware which will provide APIs to help manage your queues via your ops portal.
+
+## 2. How to use 
+
+Alton is an ASP.NET Core piece of middleware that will automatically register routes on your behalf.
+Alton is still in a preview stage and some issues may arise.
+
+Firstly, to register the middleware, you will need to add it to the service collection and register the middleware:
+
+### Registering Alton
+
+```c#
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAlton();
+    //my other services here
+}
+```
+
```

Comment:
The `IServiceCollection.AddAlton()` extension will automatically register an `SqsClientResolver` which will create an SQS client using the region parsed from the QueueUrl.

You can provide your own resolver with any delegate matching the signature:

```c#
IAmazonSQS Resolve(AltonQueueComponent queue);
```

Reply:
updated